frontend/pages/login/index.tsx

```diff
@@ -1,6 +1,7 @@
-import React, { FormEvent, useState } from 'react'
-import { EnrollmentCallToAction, EnrollmentHeader, EnrollmentInput } from '../../shared-components'
-import { CallToActionTypes, EnrollmentInputNames } from '../../models'
+import React, { FormEvent, useState } from "react"
+import { EnrollmentCallToAction, EnrollmentHeader, EnrollmentInput } from "../../shared-components"
+import { CallToActionTypes, EnrollmentInputNames } from "../../models"
+import Layout from "../../shared-components/layout/Layout"
```

Comment:
To clarify, the children I'm seeing get wrapped here are getting a unique Layout component that no other page on the site is getting?

Reply:
yes, so far the headers are different, there may not need to be a reason to wrap registration/login/passport but I felt it couldn't hurt to add for clarification that all pages a user has access to should be wrapped by Layout.